examples/src/main/resources/ai/timefold/solver/examples/vehiclerouting/vehicleRoutingSolverConfig.xml

```diff
@@ -25,63 +25,73 @@
-          </nearbySelection>
-        </destinationSelector>
-      </listChangeMoveSelector>
-      <listSwapMoveSelector>
-        <valueSelector id="2"/>
-        <secondaryValueSelector>
-          <nearbySelection>
-            <originValueSelector mimicSelectorRef="2"/>
-            <nearbyDistanceMeterClass>ai.timefold.solver.examples.vehiclerouting.domain.solver.nearby.CustomerNearbyDistanceMeter</nearbyDistanceMeterClass>
-            <nearbySelectionDistributionType>PARABOLIC_DISTRIBUTION</nearbySelectionDistributionType>
-            <parabolicDistributionSizeMaximum>40</parabolicDistributionSizeMaximum>
-          </nearbySelection>
-        </secondaryValueSelector>
-      </listSwapMoveSelector>
+      <listChangeMoveSelector />
+      <listSwapMoveSelector />
       <subListChangeMoveSelector>
         <selectReversingMoveToo>true</selectReversingMoveToo>
-        <subListSelector id="3"/>
-        <destinationSelector>
-          <nearbySelection>
-            <originSubListSelector mimicSelectorRef="3"/>
-            <nearbyDistanceMeterClass>ai.timefold.solver.examples.vehiclerouting.domain.solver.nearby.CustomerNearbyDistanceMeter</nearbyDistanceMeterClass>
-            <nearbySelectionDistributionType>PARABOLIC_DISTRIBUTION</nearbySelectionDistributionType>
-            <parabolicDistributionSizeMaximum>40</parabolicDistributionSizeMaximum>
-          </nearbySelection>
-        </destinationSelector>
       </subListChangeMoveSelector>
       <subListSwapMoveSelector>
         <selectReversingMoveToo>true</selectReversingMoveToo>
-        <subListSelector id="4"/>
-        <secondarySubListSelector>
-          <nearbySelection>
-            <originSubListSelector mimicSelectorRef="4"/>
-            <nearbyDistanceMeterClass>ai.timefold.solver.examples.vehiclerouting.domain.solver.nearby.CustomerNearbyDistanceMeter</nearbyDistanceMeterClass>
-            <nearbySelectionDistributionType>PARABOLIC_DISTRIBUTION</nearbySelectionDistributionType>
-            <parabolicDistributionSizeMaximum>40</parabolicDistributionSizeMaximum>
-          </nearbySelection>
-        </secondarySubListSelector>
       </subListSwapMoveSelector>
-      <kOptListMoveSelector>
-        <originSelector id="5"/>
-        <valueSelector>
-          <nearbySelection>
-            <originValueSelector mimicSelectorRef="5"/>
-            <nearbyDistanceMeterClass>ai.timefold.solver.examples.vehiclerouting.domain.solver.nearby.CustomerNearbyDistanceMeter</nearbyDistanceMeterClass>
-            <nearbySelectionDistributionType>PARABOLIC_DISTRIBUTION</nearbySelectionDistributionType>
-            <parabolicDistributionSizeMaximum>40</parabolicDistributionSizeMaximum>
-          </nearbySelection>
-        </valueSelector>
-      </kOptListMoveSelector>
+      <kOptListMoveSelector />
+      <!-- With Timefold Solver Enterprise Edition, uncomment the following to enable Nearby Selection: -->
```

Comment:
I see a lot of combinations got commented out as expected. However, this impacts:
a) performance tests in timefold-examples (if they run these configurations)
b) turtle tests (assuming there are some turtle tests)

Reply:
At the moment, there are  no turtle tests, so having at least some will be an improvement.
There's an issue for that already.